omod/src/main/webapp/pages/login.gsp

```diff
@@ -41,6 +37,15 @@ ${ ui.includeFragment("referenceapplication", "infoAndErrorMessages") }
             updateSelectedOption();
         });
         
+        jQuery('#login-form').submit(function(e) {
+        	var sessionLocationVal = jQuery('#sessionLocationInput').val();
+        	
+        	if (!sessionLocationVal) {
```

Comment:
@dkayiwa Oh i see. I didn't think it was necessary to do all the 3 checks. Just the one is enough. Because in javascript, null, empty string and 0 are falsy which means when you use them in place of a boolean they always return false. So there was no need doing all the checks separately. Does that make sense?